scripts/TestHarness/testers/RavenErrors.py

```diff
@@ -82,13 +87,14 @@ def check_runnable(self):
       @ In, None
       @ Out, check_runnable, boolean, If True this test can run.
     """
-    missing, too_old, _ = RavenUtils.check_for_missing_modules()
+    missing, not_qa = library_handler.checkLibraries()
```

Comment:
This is a rook requirement right now; it runs pylint on all the checkers it can find and enforces `snake_case`. I don't see a way to disable it just for RAVEN testers at this point.